.env.local.example

```
@@ -2,9 +2,31 @@ NEXT_PUBLIC_GOOGLE_ANALYTICS_ID=
 NEXT_PUBLIC_REOWN_PROJECT_ID=
 ENVIRONMENT=
 
-NEXT_PUBLIC_ALCHEMY_API_KEY=
 NEXT_PUBLIC_INFURA_API_KEY=
 
+# ==================== RPC Settings ====================
+
+# RPC Priority: Set to "ALCHEMY" to prioritize Alchemy API when both are available
+# Default (or any other value): Prioritizes individual network RPCs first
+NEXT_PUBLIC_RPC_PRIORITY="ALCHEMY"
```

dotenv-linter (4.0.0)

[warning] [QuoteCharacter] The value has quote characters (', ")

(QuoteCharacter)